### ISSUE TITLE
Crashes on spawn actor

### ISSUE THREAD

**Issue by Gatts011** [npcbx/AsyncFuncCaller#1]
Assertion failed: IsInGameThread() [File:D:\UnrealEngine\Engine\Source\Runtime\Engine\Classes\Animation/AnimInstance.h] [Line: 1367] 

(4.25)

**Comment by npcbx** [comment]
yes it is.Spawn Actor and any other work that can influence game world or ui can only be done in the game thread.
